### PR TITLE
[GAM] Updating readme with new results

### PR DESCRIPTION
**Pull request by otiliastr**
Updating readme file and run configurations with new results and their corresponding run configurations, after the bug fix in PR #82 .

### CODE REVIEW

File: research/gam/README.md [hunk 1]
```diff
@@ -40,6 +40,20 @@ More details can be found in our
 [slides](https://drive.google.com/open?id=1tWEMoyrbLnzfSfTfYFi9eWgZWaPKF3Uu) or
 [poster](https://drive.google.com/file/d/1BZNR4B-xM41hdLLqx4mLsQ4KKJOhjgqV/view).
 
+## Updated Results
+A bug was discovered in the implementation of the GAM agreement regularization term after publication. We have fixed the bug (PR #82) and have rerun the affected experiments. Below are the updated results (note that the GAM* results are not affected). 
+
+Dataset  | Method      | Updated Accuracy (mean ± stderr)
+-------- | :---------: | :-------------: 
+Cora     | MLP + GAM   | 80.2 ± 0.31
```
Comment (arjung) [inline research/gam/README.md:48]: Which rows do these correspond to in Table 1 of the paper (in other words, what's the hidden layer configuration)? That'll help the reader understand the difference(s). Optionally, you may also add a column here listing the old values from the paper.

File: research/gam/README.md [hunk 2]
```diff
@@ -40,6 +40,20 @@ More details can be found in our
...
+Dataset  | Method      | Updated Accuracy (mean ± stderr)
+-------- | :---------: | :-------------: 
+Cora     | MLP + GAM   | 80.2 ± 0.31 
+&nbsp;   | GCN + GAM   | 84.8 ± 0.06 
+Citeseer | MLP + GAM   | 73.2 ± 0.06
+&nbsp;   | GCN + GAM   | 72.2 ± 0.44
+Pubmed   | MLP + GAM   | 75.6 ± 0.07
+&nbsp;   | GCN + GAM   | 81.0 ± 0.09
+
+Although some of these numbers are lower than what was originally reported, the takeaways presented in our paper still hold: GAM adds a significant boost to the original base models, and also performs better than other forms of regularization reported in our paper. Nevertheless, we appologise for any inconvenience caused by this bug!
```
Comment (arjung) [inline research/gam/README.md:55]: s/appologise/apologize

File: research/gam/gam/trainer/trainer_cotrain.py [hunk 3]
```diff
@@ -355,12 +355,15 @@ def _select_samples_to_label(self, data, trainer_cls, session):
         assign to each of the selected nodes.
     """
     # Select the candidate samples for self-labeling, and make predictions.
-    # Remove the validation samples from the unlabeled data, if there, to avoid
-    # self-labeling them.
+    # Remove the validation and test samples from the unlabeled data, if there,
```
Comment (arjung) [inline research/gam/gam/trainer/trainer_cotrain.py:358]: if there -> if they exist ?

File: research/gam/gam/trainer/trainer_cotrain.py [hunk 4]
```diff
@@ -355,12 +355,15 @@ def _select_samples_to_label(self, data, trainer_cls, session):
...
+    # to avoid self-labeling them. We could potentially leave the test edges
+    # but once a node self-labeled, is label is fixed for the remaining
```
Comment (arjung) [inline research/gam/gam/trainer/trainer_cotrain.py:360]: Looks like this line has typos.

**Review by arjung — CHANGES_REQUESTED** [review]
Thanks for the PR, Otilia!

File: research/gam/gam/trainer/trainer_cotrain.py (hunk 4 above)
Comment (otiliastr) [reply research/gam/gam/trainer/trainer_cotrain.py]: Thanks for catching this! I've fixed it.

File: research/gam/gam/trainer/trainer_cotrain.py (hunk 3 above)
Comment (otiliastr) [reply research/gam/gam/trainer/trainer_cotrain.py]: Thanks! I've changed it.

File: research/gam/README.md (hunk 1 above)
Comment (otiliastr) [reply research/gam/README.md]: Thanks for the suggestion! I updated the readme with a new table similar to the one in the paper, comparing the new results with the competing methods. Markdown doesn't support nice table formatting options, so I added it as an image. See my latest commit.

**Review by arjung — APPROVED** [review]
Thanks for the updates! Looks great.

**Pull request merged**